blackbox/requirements.txt

```diff
@@ -4,14 +4,7 @@
 crash
 crate
 crate-docs-theme
-psycopg2-binary==2.7.5
```

Comment:
is this change related to the license update, or just a cleanup?

Reply:
it's a cleanup. but I've included it in this change, because if I don't clean up these modules, I will have to document their licensing too (which is unnecessary)

Reply:
`psycopg2-binary` is required for the logic tests. We can't remove it.